packages/utils/src/generateBlogPostStructuredData.ts

```diff
@@ -0,0 +1,49 @@
+import { SCHEMA_CONTEXT_URL } from './constants/structuredDataConstants'
+
+import type { BlogPosting, Organization, WithContext } from 'schema-dts'
+
+type GenerateBlogPostStructuredDataProps = {
```

Comment:
@CharlyMartin - Would you refactor this? Would you use something like `type GenerateBlogPostStructuredDataProps = Partial<BlogPosting> & {`?

Reply:
Thank you both!